README.md

```diff
@@ -1,33 +1,46 @@
-# AWESOME TAURI
+<!--lint disable awesome-heading awesome-github awesome-toc double-link -->
 
-This is where we collect all of the best stuff from the ecosystem and community. To contribute, just make a PR.
+<h2 align='center'>Awesome Tauri</h2>
 
+<p align='center'>
+This is where we collect all of the best stuff from the ecosystem and community.
+<br><br>
 
-## Apps
-| App Name | Repo | Website | Description |
-| - | - | - | - |
-| | | |
+<a href='https://awesome.re'>
+<img src='https://awesome.re/badge-flat.svg' alt='Awesome'>
+</a>
+</p>
+
+## Resources
+
+- [Homepage](https://tauri.studio)
+- [GitHub Repo](https://github.com/tauri-apps/tauri)
+
+## Get Started
+
+- [create-tauri-app](https://github.com/tauri-apps/tauri/tree/next/tooling/create-tauri-app) - Rapidly scaffold your Tauri app.
+
+### Templates
+
+### Tutorials
+
+### Blog Articles
 
 ## Plugins
-| Plugin Name | Repo | Description |
-|-|-|-|
-| tauri-plugin-positioner | https://github.com/JonasKruckenberg/tauri-plugin-positioner | Helps positioning Tauri windows at known locations |
-| | |
-
-## Templates
-| Template Name | Repo | Description |
-| - | - | - |
-| | |
-
-## Tutorials
-| Tutorial Name | Link | Description |
-| - | - | - |
-| | |
-
-## Blog Articles
-| Article Name | Link | Description |
-| - | - | - |
-| | |
-
-# License
-All content in this repo is licensed under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+### Official Plugins
```

Comment:
I think official and community plugins should all be listed under one category "Plugins" and the official plugins will be listed with an official badge next to it.

Reply:
I like it! This
![official](https://img.shields.io/badge/-official-blue)
or this
![official](https://img.shields.io/badge/-official-brightgreen)
?

Reply:
blue, to go along with the tauri brand.

Reply:
wait a sec, let's try these: actually the tauri brand colors:
![official](https://img.shields.io/badge/-official-FFC131)
![official](https://img.shields.io/badge/-official-24C8DB)
![official](https://img.shields.io/badge/-official-F6F6F6)
![official](https://img.shields.io/badge/-official-0F0F0F)

Reply:
oh man that orange feels so sexy for some reason.

Reply:
or the cyan one with dark text too.

Reply:
I think you can't influence the text color with shields.io 🤔 so orange it is 😄